controllers/preflightvalidation_reconciler.go

```diff
@@ -105,6 +107,10 @@ func (r *PreflightValidationReconciler) Reconcile(ctx context.Context, req ctrl.
 		log.Info("PreflightValidation reconciliation success")
 		return ctrl.Result{}, nil
 	}
+	// if not all the modules has been reconciled, then maybe there is a problem with build
+	// configuration. Since build configuration is stored in the ConfigMap, and we cannot watch
+	// ConfigMap (since we don't know which ones to watch), then we need a requeue in order to run
```

Comment:
> since we don't know which ones to watch

How so? We know the name of the configmap from the module no?!

Reply:
yes, but that can only be filtered after the event has been raised. You cannot filter out ConfigMap before event has been raised. Since there can be a lot of ConfigMap in the cluster, we don't want to start filter each time